core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -520,21 +520,22 @@ class GroupMetadataManager(brokerId: Int,
   def scheduleLoadGroupAndOffsets(offsetsPartition: Int, onGroupLoaded: GroupMetadata => Unit): Unit = {
     val topicPartition = new TopicPartition(Topic.GROUP_METADATA_TOPIC_NAME, offsetsPartition)
     if (addLoadingPartition(offsetsPartition)) {
+      val nowInMs = time.milliseconds()
```

Comment:
nit: move it after info line. Maybe get rid of variable altogether and put is on the method call itself.

Reply:
Makes sense, done.